.github/ISSUE_TEMPLATE/3.pro-support.yml

```diff
@@ -39,6 +39,25 @@ body:
   - type: textarea
     attributes:
       label: The problem in depth 🔍
+  - type: textarea
+    attributes:
+      label: Steps to reproduce 🕹
+      description: |
+        Provide a link to a live example (you can use codesandbox.io) and an unambiguous set of steps to reproduce this bug.
+        Include code to reproduce, if relevant (which it most likely is).
+
+        You can use the official codesandbox template of the [DataGrid](https://codesandbox.io/s/datagrid-v5-quick-start-54iz1) or [DataGridPro](https://codesandbox.io/s/datagridpro-v5-quick-start-5pjhc) as a starting point.
+
+        If you have an issue concerning TypeScript please start from this TypeScript playground: https://mui.com/r/ts-issue-template.
```

Comment:
The target seems to be specific to `@material-ui/core` (i.e. outdated, and not relevant to DataGrid)

Reply:
Fixed in https://github.com/mui-org/material-ui/pull/30596.